package.json

```diff
@@ -129,7 +129,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "tsc --noEmit && lint-staged",
-      "pre-push": "yarn danger local --failOnErrors"
+      "pre-push": "yarn danger local -b main --failOnErrors"
```

Comment:
is running `yarn danger local -b main --failOnErrors` locally supposed to work for me at this point?  

I'm getting the `Error: fatal: ambiguous argument 'master...HEAD': unknown revision or path not in the working tree.` error

Reply:
No, there hasn't been any more feedback on my [bug report](https://github.com/danger/danger-js/issues/1057).  Maybe I'll just open a PR